tests/network/vmi_istio.go

```diff
@@ -496,10 +488,6 @@ var _ = SIGDescribe(" Istio with masquerade binding", decorators.Istio, Serial,
 
 var _ = SIGDescribe(" Istio with passt binding", decorators.Istio, decorators.NetCustomBindingPlugins, Serial, istioTestsWithPasstBinding)
 
-func istioServiceMeshDeployed() bool {
-	return strings.ToLower(os.Getenv(istioDeployedEnvVariable)) == "true"
```

Comment:
Do we still need `istioDeployedEnvVariable`?

Reply:
Thanks for the heads up, I removed it.